src/Main.java

```diff
@@ -1,5 +1,112 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        // задание 1
+        {
+            byte a = 1;
+            short b = 18000;
+            int c = 120000000;
+            long d = 1000000000000L;
+            float e = 5.5f;
+            double f = 258.655878565;
+            System.out.println("Значение переменной " + " a " + " с типом byte равно " + a);
+            System.out.println("Значение переменной " + " b " + " с типом short равно " + b);
+            System.out.println("Значение переменной " + " c " + " с типом int равно " + c);
+            System.out.println("Значение переменной " + " d " + " с типом long равно " + d);
+            System.out.println("Значение переменной " + " e " + " с типом float равно " + e);
+            System.out.println("Значение переменной " + " f " + " с типом double равно " + f);
+        }
+        // задание 2
+        {
+            byte a1 = 67;
+            short b1 = -159;
+            int c1 = 569;
+            long d1 = 987678965549L;
+            float e1 = 2.786f;
+            double f1 = 27.12;
+            char g1 = 27897;
+        }
+        // задание 3
+        {
+            int group1 = 23;
+            int group2 = 27;
+            int group3 = 30;
+            int students = group1 + group2 + group3;
+            int sheets = 480;
+            int sheetsForEachStudent = sheets / students;
+            System.out.println("На каждого ученика рассчитано " + sheetsForEachStudent + " листов бумаги");
+        }
+        // задание 4
+        {
+            int productivityIn2Minutes = 16;
+            int productivityIn1Minutes = productivityIn2Minutes / 2;
+            int productivityIn20Minutes = productivityIn1Minutes * 20;
+            int productivityInDay = productivityIn1Minutes * 1440;
+            int productivityIn3Days = productivityIn1Minutes * 4320;
```

Comment:
никогда не нужно считать в уме, иначе смысл писать программы, если ты считаешь в уме? Можно было бы всё посчитать в уме)
```suggestion
            int productivityIn3Days = 3 * productivityInDay;
```